### PR TITLE
Add details of planned tests to the homepage and planned tests page

### PR DESCRIPTION
**Pull request by quis**
image
image

### CODE REVIEW

File: src/planned-tests.html [hunk 1]
```diff
@@ -67,14 +67,28 @@ <h2 class="govuk-heading-m govuk-!-margin-bottom-2">
...
       </h2>
       <h3 class="govuk-body govuk-!-margin-bottom-6">
-        Reading
+        Reading, Berkshire
       </h3>
       <p class="govuk-body">
         The UK government will send a test alert.
       </p>
       <p class="govuk-body">
         If you get an alert, your phone or tablet may make a loud siren-like sound. You do not need to take any action.
```
Comment (benthorner) [inline src/planned-tests.html:79]: I notice the alert above has an Android caveat. 

I'm guessing we expect all Apple devices to have updated by this time.

**Review by benthorner — APPROVED** [review]
📯

File: src/planned-tests.html (hunk 1 above)
Comment (quis) [reply src/planned-tests.html:79]: Only Android devices can receive alerts sent on the test or operator channels.

Adoption of iOS 14.5 or above (i.e. iPhones that will receive alerts) is now at 76% (according to traffic to GOV.UK). I think this is high enough that we don’t need to caveat it.

Comment (benthorner) [reply src/planned-tests.html:79]: Ah, I didn't know that. Thanks for explaining.

**Pull request merged**